executor.go

```diff
@@ -600,8 +591,8 @@ func (e *Executor) executeTopNSlice(ctx context.Context, index string, c *pql.Ca
 		N:                 int(n),
 		Src:               src,
 		RowIDs:            rowIDs,
-		FilterName:        field,
-		FilterValues:      filters,
+		FilterName:        attrName,
```

Comment:
Do we want to keep these old names here?

Reply:
Not necessarily. I thought I would leave them until we decided that `attrName` was good.